Sources/Ledger/PartialTxOut.swift

```diff
@@ -51,12 +51,12 @@ extension PartialTxOut {
     init?(_ txOutRecord: FogView_TxOutRecord, viewKey: RistrettoPrivate) {
         guard let targetKey = RistrettoPublic(txOutRecord.txOutTargetKeyData),
               let publicKey = RistrettoPublic(txOutRecord.txOutPublicKeyData),
-              [0,4,8].contains(txOutRecord.txOutAmountMaskedTokenID.count),
+              [0, 4, 8].contains(txOutRecord.txOutAmountMaskedTokenID.count),
               let commitment = TxOutUtils.reconstructCommitment(
-                                                    maskedValue: txOutRecord.txOutAmountMaskedValue,
-                                                    maskedTokenId: txOutRecord.txOutAmountMaskedTokenID,
-                                                    publicKey: publicKey,
-                                                    viewPrivateKey: viewKey),
+                                            maskedValue: txOutRecord.txOutAmountMaskedValue,
+                                            maskedTokenId: txOutRecord.txOutAmountMaskedTokenID,
+                                            publicKey: publicKey,
+                                            viewPrivateKey: viewKey),
```

Comment:
while beaking the conditiosn to the next line is more readable Im not sure it makes sense in every instance, it also adds an extra line to the file. I think in this case it does look better so I will make this change, but the identation on the function parameters I like being atleast 2 because the function name is so long and it improve readability. will commit this change separately.